docs/contributor/count-contributions.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Correct your information for better contribution
+---
+
+## Check Your contributions in the dashboard
+After contributing to [karmada-io](https://github.com/karmada-io), including issues, comments, pull requests, etc. You can check your contributions [Here](https://karmada.devstats.cncf.io/d/66/developer-activity-counts-by-companies).
+
+## Ensure the information in the company column is correct
+Once you find the information in the company column is empty or wrong, we strongly recommend that you fix it.  
+Here is an example, the company should be `Huawei Technologies Co. Ltd` instead of `HUAWEI`:
```

Comment:
How do I know my company name in CNCF systems? Do we have a guide?

Reply:
Added